code/modules/spells/spell_types/mime.dm

```diff
@@ -27,6 +27,80 @@
 		invocation_type ="none"
 	..()
 
+/obj/effect/proc_holder/spell/aoe_turf/conjure/mime_chair
+	name = "Invisible Chair"
+	desc = "The mime's performance transmutates a chair into physical reality."
+	school = "mime"
+	panel = "Mime"
+	summon_type = list(/obj/structure/chair/mime)
+	invocation_type = "emote"
+	invocation_emote_self = "<span class='notice'>You conjure an invisible chair and sit down.</span>"
+	summon_lifespan = 250
+	charge_max = 300
+	clothes_req = FALSE
+	antimagic_allowed = TRUE
+	range = 0
+	cast_sound = null
+	human_req = TRUE
+
+	action_icon_state = "mime"
+	action_background_icon_state = "bg_mime"
+
+/obj/effect/proc_holder/spell/aoe_turf/conjure/mime_chair/Click()
+	if(usr && usr.mind)
+		if(!usr.mind.miming)
+			to_chat(usr, "<span class='notice'>You must dedicate yourself to silence first.</span>")
+			return
+		invocation = "<B>[usr.real_name]</B> pulls out an invisible chair and sits down."
+	else
+		invocation_type ="none"
+	..()
+
+/obj/effect/proc_holder/spell/aoe_turf/conjure/mime_chair/cast(list/targets,mob/user = usr)
+	..()
+	var/turf/T = user.loc
+	for (var/obj/structure/chair/A in T)
+		if (is_type_in_list(A, summon_type))
+			A.setDir(user.dir)
+			A.buckle_mob(user)
+
+/obj/effect/proc_holder/spell/aoe_turf/conjure/mime_box
+	name = "Invisible Box"
+	desc = "The mime's performance transmutates a box into physical reality."
+	school = "mime"
+	panel = "Mime"
+	summon_type = list(/obj/item/storage/box/mime)
+	invocation_type = "emote"
+	invocation_emote_self = "<span class='notice'>You conjure up an invisible box, large enough to store a few things.</span>"
+	summon_lifespan = 250
+	charge_max = 300
+	clothes_req = FALSE
+	antimagic_allowed = TRUE
+	range = 0
+	cast_sound = null
+	human_req = TRUE
+
+	action_icon_state = "mime"
+	action_background_icon_state = "bg_mime"
+
+/obj/effect/proc_holder/spell/aoe_turf/conjure/mime_box/cast(list/targets,mob/user = usr)
+	..()
+	var/turf/T = user.loc
+	for (var/obj/item/storage/box/mime/B in T)
+		user.put_in_hands(B)
+		B.alpha = 255
+		addtimer(CALLBACK(B, /obj/item/storage/box/mime/.proc/emptyStorage, FALSE), (summon_lifespan - 1))
+
+/obj/effect/proc_holder/spell/aoe_turf/conjure/mime_box/Click()

```

Comment:
why are you overriding click?

Reply:
for the same reason the mime wall spell does